### PR TITLE
refactor(cli): update docs for CLI changes

### PR DESCRIPTION
**Pull request by jbrockopp**
We've cut a `v0.5.0` release candidate but probably should wait to merge until we cut the official release 👍

### CODE REVIEW

File: content/cli/config/generate.md [hunk 1]
```diff
@@ -19,16 +19,17 @@ For more information, you can run `vela generate config --help`.
...
-| `token`         | user token from server       | `N/A`       |
-| `api-version`   | API version for server       | `N/A`       |
-| `log-level`     | change the CLI logging level | `N/A`       |
-| `org`           | name of organization         | `N/A`       |
-| `repo`          | name of repository           | `N/A`       |
-| `secret-engine` | name of secret backend       | `N/A`       |
-| `secret-type`   | name of secret type          | `N/A`       |
+| Name          | Description                         | Environment Variables                    |
+| ------------- | ----------------------------------- | ---------------------------------------- |
+| `addr`        | full URL to API server              | `VELA_ADDR`, `CONFIG_ADDR`               |
```
Comment (kneal) [inline content/cli/config/generate.md]: I believe this should be `addr` to `api.addr`

File: content/cli/config/generate.md [hunk 2]
```diff
@@ -19,16 +19,17 @@ For more information, you can run `vela generate config --help`.
...
+| `token`       | user token from API server          | `VELA_TOKEN`, `CONFIG_TOKEN`             |
```
Comment (kneal) [inline content/cli/config/generate.md]: I believe this should be `token` to `api.token`

File: content/cli/service/view.md [hunk 3]
```diff
@@ -19,19 +19,20 @@ For more information, you can run `vela view service --help`.
...
-| `repo`    | name of repository   | `BUILD_REPO`     |
-| `build`   | number of build      | `BUILD_NUMBER`   |
-| `service` | number of service    | `SERVICE_NUMBER` |
-| `output`  | format the output    | `N/A`            |
+| Name     | Description                       | Environment Variables        |
+| -------- | --------------------------------- | ---------------------------- |
+| `org`    | name of organization for the service | `VELA_ORG`, `SERVICE_ORG`       |
+| `repo`   | name of repository for the service   | `VELA_REPO`, `SERVICE_REPO`     |
+| `build`  | number of build for the service      | `VELA_BUILD`, `SERVICE_BUILD`   |
+| `service`   | number of the service                | `VELA_SERVICE`, `SERVICE_NUMBER`   |
```
Comment (kneal) [inline content/cli/service/view.md]: Think the table formatting might be off here

**Review by kneal — CHANGES_REQUESTED** [review]

**Review by KellyMerrick — APPROVED** [review]
lgtm

**Review by plyr4 — APPROVED** [review]

**Pull request merged**